tests/acceptance/test_conftest.py

```diff
@@ -22,17 +22,8 @@
 
```

Comment:
Because this is in fact testing that the `login` fixture did its job, and nothing else.